defi/src/protocols/data3.ts

```diff
@@ -27303,7 +27303,7 @@ const data3: Protocol[] = [
     cmcId: null,
     category: "Lending",
     chains: ["Solana"],
-    oracles: ["TWAP", "Pyth"],
+    oracles: ["TWAP", "Pyth", "Switchboard"],
```

Comment:
What markets are using  Switchboard ?

Reply:
<img width="499" alt="Screenshot 2024-11-04 at 10 37 55 AM" src="https://github.com/user-attachments/assets/3d137df7-6c0f-4a23-b57d-3adc3b2f45bb">
<img width="1728" alt="Screenshot 2024-11-04 at 10 37 57 AM" src="https://github.com/user-attachments/assets/4b99d1f9-e8d2-4187-8ca3-fd3f317547e8">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 02 AM" src="https://github.com/user-attachments/assets/571939c3-fae6-4656-80d2-bda36398295f">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 07 AM" src="https://github.com/user-attachments/assets/b9857b4b-06d5-4467-ba15-65e1cdd8b3c8">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 12 AM" src="https://github.com/user-attachments/assets/edd8ef73-808b-4459-95b2-8f433e558a42">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 15 AM" src="https://github.com/user-attachments/assets/cf480fe4-0865-4beb-92f6-e92c5d516951">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 20 AM" src="https://github.com/user-attachments/assets/97939343-c019-495e-8076-71763bb2b0a1">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 24 AM" src="https://github.com/user-attachments/assets/6cdf7b96-e1db-43fd-971e-97e41d9cf6e5">
<img width="1728" alt="Screenshot 2024-11-04 at 10 38 28 AM" src="https://github.com/user-attachments/assets/f10c49ff-a771-4723-b049-8774dab97d50">

Reply:
WEN
MOUTAI
ZEUS
BODEN
SHDW
strongSOL
ISC
KMNO 
POPCAT
MANEKI
BONKSOL
JUPSOL
JUCYSOL
COMPASSSOL
lanternSOL
stakeSOL
pwrSOL
DRIFT
hubSOL
MOTHER
dSOL
hSOL
ZEX
BILLY
tremp
vSOL
pathSOL
CLOUD
SEND
ATLAS
POLIS
BLOCK
FWOG
SCS
BNSOL
ORE
sSOL
MOODENG
SPX
GIGA
mini
$michi 
DBR
GOAT
GRASS
GMT

Reply:
(All the tokens denoted with the switchboard logo on their site)

Reply:
What % of TVL is secured by Switchboard? On those markets

Reply:
hey @Define101, I can confirm that we use Switchboard oracles to secure all tokens listed by @mgild. Since we operate with cross-collateralization, our entire TVL (currently 2.8M) is safeguarded through Switchboard oracles.

Reply:
@eldios 9.1m$ is less than 5% of the total TVL.

Reply:
@solwhitey we added the oracle to rain fi here https://github.com/DefiLlama/defillama-server/pull/8524

Reply:
For marginfi is only securing 9.1m$ in a 460m$ TVL. Very small %.